server/src/resolvers/inputTypes.ts

```diff
@@ -25,12 +26,21 @@ export const FilterInput = inputObjectType({
   },
 })
 
+export const OwnerInput = inputObjectType({
+  name: 'OwnerInput',
+  definition(t) {
+    t.nonNull.field('id', { type: 'String' })
```

Comment:
So the client has to know the owner ids?
Why not using the name + sourceId to connect to the right Owner?